packages/cli/src/lib/config.ts

```diff
@@ -76,6 +80,7 @@ export async function loadCliConfig(options: Options) {
     experimentalEnvFunc: options.mockEnv
       ? async name => process.env[name] || 'x'
       : undefined,
+    cliOptions: options.cliOptions,
```

Comment:
To clarify the later comment about avoiding the changes to `config-loader`, let's create the extra config object here instead and add it to the returned `appConfigs`.